modules/geth/charts.go

```diff
@@ -0,0 +1,182 @@
+		},
+	}
+	chartTxPoolCurrent = Chart{
+		ID: "txpoolcurrent",
+		Title: "Transaction Pool",
+		Units: "transactions", 
+		Fam: "tx_pool",
+		Ctx: "geth.tx_pool_current", 
+		Dims: Dims{
+			{ID: txPoolInvalid, Name: "Invalid transaction pool"},
+			{ID: txPoolPending, Name: "Pending transaction pool"},
+			{ID: txPoolLocal, Name: "Local transaction pool"},
+			{ID: txPoolNofunds, Name: "Pool of transactions with no funds" },
+		},
+	}
+	chartTxPoolQueued = Chart{
+		ID: "txpoolqueued",
+		Title: "Queued Transaction Pool",
+		Units: "transactions", 
+		Fam: "tx_pool",
+		Ctx: "geth.tx_pool_queued", 
+		Dims: Dims{
+			{ID: txPoolQueuedDiscard, Name: "discard"},
+			{ID: txPoolQueuedEviction, Name: "eviction"},
+			{ID:txPoolQueuedNofunds, Name: "no_funds" },
+			{ID: txPoolQueuedRatelimit, Name: "ratelimit"},
+		},
+	}
+	chartP2PNetwork = Chart{
+		ID: "p2p_network", 
+		Title: "P2P bandwidth",
+		Units: "bytes", 
+		Fam: "p2p_bandwidth", 
+		Ctx: "geth.p2p_bandwidth",
+		Dims: Dims{
+			{ID: p2pIngress, Name: "ingress"},
+			{ID: p2pEgress, Name: "egress", Mul: -1},
+		},
+	}
+	chartP2PNetworkDetails = Chart{
+		ID: "p2p_eth_65", 
+		Title: "Eth/65 Network utilization",
+		Units: "bytes", 
+		Fam: "p2p_eth_65",
+		Ctx: "geth.p2p_eth_65",
+		Dims: Dims{
+			{ID: p2pIngressEth650x00, Name: "Eth/65 handshake ingress"},
+			{ID: p2pIngressEth650x01, Name: "Eth/65 new block hash ingress"},
+			{ID: p2pIngressEth650x03, Name: "Eth/65 block header request ingress"},
+			{ID: p2pIngressEth650x04, Name: "Eth/65 block header response ingress"},
+			{ID: p2pIngressEth650x05, Name: "Eth/65 block body request ingress"},
+			{ID: p2pIngressEth650x06, Name: "Eth/65 block body response ingress"},
+			{ID: p2pIngressEth650x08, Name: "Eth/65 transactions announcement ingress"},
+			{ID: p2pEgressEth650x00, Name: "Eth/65 handshake egress", Mul: -1},
+			{ID: p2pEgressEth650x01, Name: "Eth/65 new block hash egress", Mul: -1},
+			{ID: p2pEgressEth650x03, Name: "Eth/65 block header request egress", Mul: -1},
+			{ID: p2pEgressEth650x04, Name: "Eth/65 block header response egress", Mul: -1},
+			{ID: p2pEgressEth650x05, Name: "Eth/65 block body request egress", Mul: -1},
+			{ID: p2pEgressEth650x06, Name: "Eth/65 block body response egress", Mul: -1},
+			{ID: p2pEgressEth650x08, Name: "Eth/65 transactions announcement egress", Mul: -1},
+		},
+	}
+	chartNumberOfPeers = Chart{
+		ID: "p2p_general_info",
+		Title: "Number of Peers",
+		Units: "peers",
+		Fam: "p2p_peers",
+		Ctx: "geth.p2p_peers",
+		Dims: Dims{
+			{ID: p2pPeers, Name: "Peers"},
+		},
+	}
+	chartRpcInformation = Chart{
+		ID: "rpc_metrics", 
+		Title: "RPC information", 
+		Units: "rpc calls",
+		Fam: "rpc_metrics",
+		Ctx: "geth.rpc_metrics",
+		Dims: Dims{
+			{ID: rpcFailure, Name: "Failed RPC requests", Algo: "percentage-of-absolute-row"},
+			{ID: rpcSuccess, Name: "Successful RPC requests", Algo: "percentage-of-absolute-row"},
+		},
+	}
+)
```

Comment:
```suggestion
)

```

Fixes _No newline at end of file_